scoring/scoring_utils.py

```diff
@@ -102,7 +103,7 @@ def get_trials_df_dict(logfile):
     """
   trials_dict = get_trials_dict(logfile)
   trials_df_dict = {}
-  for trial in trials_dict:
+  for trial in trials_dict.items():
```

Comment:
is this supposed to be items() or keys()?

Reply:
yup, oops that is supposed to be keys(). Thanks for catching that.